src/josepy/testdata/ec_p256_key.pem

```diff
@@ -0,0 +1,8 @@
+-----BEGIN EC PARAMETERS-----
+BggqhkjOPQMBBw==
+-----END EC PARAMETERS-----
+-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIMUXjUASqqk7YRvKE5gXYaeBEGCJkitXVan2tssjJ9q3oAoGCCqGSM49
+AwEHoUQDQgAEjjQtV+fA7J/tK8dPzYq7jRPNjF8r5p6LW2R25S2Gw5UQ8DDz/zPs
+9gqwcfqGUZKWxbEWgWXv7S8zRBEZuae8Jw==
+-----END EC PRIVATE KEY-----
```

Comment:
Just out of curiosity, how did you generate these keys? I think the `README` file should be updated with the command.

Reply:
I generated them using OpenSSL
`openssl ecparam -name prime256v1 -genkey -out ec_p256_key.pem`
`openssl ecparam -name secp384r1 -genkey -out ec_p384_key.pem`
`openssl ecparam -name secp521r1 -genkey -out ec_p521_key.pem`

I will update the file `testdata/README` accordingly.

Reply:
Fixed by 2f8bebeb